recipe/shopware.php

```diff
@@ -105,42 +105,18 @@
 
 function getPlugins(): array
 {
-    $output = explode("\n", run('cd {{release_path}} && {{bin/console}} plugin:list'));
-
-    // Take line over headlines and count "-" to get the size of the cells.
-    $lengths = array_filter(array_map('strlen', explode(' ', $output[4])));
-    $splitRow = function ($row) use ($lengths) {
-        $columns = [];
-        foreach ($lengths as $length) {
-            $columns[] = trim(substr($row, 0, $length));
-            $row = substr($row, $length + 1);
-        }
-        return $columns;
-    };
-    $headers = $splitRow($output[5]);
-    $splitRowIntoStructure = function ($row) use ($splitRow, $headers) {
-        $columns = $splitRow($row);
-        return array_combine($headers, $columns);
-    };
-
-    // Ignore first seven lines (headline, title, table, ...).
-    $rows = array_slice($output, 7, -3);
-
-    $plugins = [];
-    foreach ($rows as $row) {
-        $pluginInformation = $splitRowIntoStructure($row);
-        $plugins[] = $pluginInformation;
-    }
+    $output = explode("\n", run('cd {{release_path}} && {{bin/console}} plugin:list --json'));
```

Comment:
Maybe I'm not awake yet, but explode returns an array and json_decode expectes a string? 🤔

Reply:
@Schrank probably copy&paste error 🤦...give me some minutes, will verify and probably fix  😅